python/test/t_MarginalUniformOrderStatistics_std.py

```diff
@@ -0,0 +1,88 @@
+#! /usr/bin/env python
+
+import openturns as ot
+import openturns.experimental as otexp
+import openturns.testing as ott
+
+ot.TESTPREAMBLE()
+
+# Instantiate one distribution object
+distribution = otexp.MarginalUniformOrderStatistics(4, [1, 3])
+print("Distribution ", repr(distribution))
+print("Distribution ", distribution)
+
+# Is this distribution elliptical ?
+print("Elliptical = ", distribution.isElliptical())
+
+# Is this distribution continuous ?
+print("Continuous = ", distribution.isContinuous())
+
+# Test for realization of distribution
+oneRealization = distribution.getRealization()
+print("oneRealization=", oneRealization)
+
+# Test for sampling
+size = 10000
+oneSample = distribution.getSample(size)
+print("oneSample first=", oneSample[0], " last=", oneSample[size - 1])
+print("mean=", oneSample.computeMean())
+print("covariance=", oneSample.computeCovariance())
```

Comment:
also distribution validation already checks if emprirical moments match the ones from the distribution, so this block can be removed

Reply:
Fixed